docs/pages/eas-update/debug.mdx

```diff
@@ -129,166 +131,15 @@ If the channel is not linked to the branch we expect, we can change the link wit
-```
-
-### Viewing all assets included in an update
-
-It may be helpful to see which assets are included in our update bundle. We can see a list of named assets by running:
-
-<Terminal cmd={['$ npx expo export']} />
-
-### Debugging of native code while loading the app through expo-updates
-
-By default, we need to make a release build for `expo-updates` to be enabled and to load updates rather than reading from a development server. This is because debug builds behave like normal React Native project debug builds.
-
-To make it easier to test and debug native code in an environment that is closer to production, follow the steps below to create a debug build of the app with `expo-updates` enabled.
-
-We also provide a [step-by-step guide to try out EAS Update quickly](/eas-update/build-locally) in a local development environment using Android Studio or Xcode, with either release or debug builds of the app.
-
-#### iOS local builds
-
-- Set the debug environment variable: `export EX_UPDATES_NATIVE_DEBUG=1`
-- Reinstall pods with `npx pod-install`. The `expo-updates` podspec now detects this environment variable, and makes changes so that the debug code that would normally load from the Metro packager is bypassed, and the app is built with the EXUpdates bundle and other dependencies needed to load updates from EAS.
-- [Ensure the desired channel is set in your **Expo.plist**](/bare/updating-your-app/#configuring-the-channel-manually)
-- Modify the application Xcode project file to force bundling of the application JavaScript for both release and debug builds:
-
-```
-sed -i '' 's/SKIP_BUNDLING/FORCE_BUNDLING/g;' ios/&lt;project name&gt;.xcodeproj/project.pbxproj
-```
-
-- Execute a [debug build](/debugging/runtime-issues/#native-debugging) of the app with Xcode or from the command line.
-
-#### Android local builds
-
-- Set the debug environment variable: `export EX_UPDATES_NATIVE_DEBUG=1`
-- [Ensure the desired channel is set in your **AndroidManifest.xml**](/bare/updating-your-app/#configuring-the-channel-manually)
-- Execute a [debug build](/debugging/runtime-issues/#native-debugging) of the app with Android Studio or from the command line.
-
-#### EAS Build
-
-Alternatively, we can use EAS to create a debug build where `expo-updates` is enabled. The environment variable is set in **eas.json**, as shown in the example below:
-
-```json eas.json
-{
-  "build": {
-    "preview_debug": {
-      "env": {
-        "EX_UPDATES_NATIVE_DEBUG": "1"
-      },
-      "android": {
-        "distribution": "internal",
-        "withoutCredentials": true,
-        "gradleCommand": ":app:assembleDebug"
-      },
-      "ios": {
-        "simulator": true,
-        "buildConfiguration": "Debug"
-      },
-      "channel": "preview_debug"
-    }
-  }
-}
-```
+After publishing, the output will display the branch and the runtime version. This information can help us verify that we're creating an update with the configuration we expect.
 
 ## Mitigation steps
```

Comment:
This is now a bit disconnected from the previous section. Any 'root cause of the issue' we might have found, was a configuration one, not an issue with the actual update up to this point.